### PR TITLE
contributor aws account creation

### PR DESCRIPTION
**Pull request by amnonkhen**
We will go with an "aws account per contributor" approach.
I changed the docs to include the necessary commands.
If necessary, we can automate the email notification to the contributor.

Ticket #277

### CODE REVIEW

**Review by Wkt8 — COMMENTED** [review]
I'm guessing that we'd need aws cli installed in order to create an AWS user for the contributor. Do we need any permissions / config to be able to add the aws user to the hca-contributor group?

**Comment by mshadbolt** [comment]
I have the same question as Wei, basically the Wranglers currently share a set of 'wrangler' keys. Do these keys have the right permissions in order to run the commands to create users as given in the doc?

Should there be a way to keep track of who we have given each key?

**Comment by amnonkhen** [comment]
Accounts and their keys are accessible in the AWS console.
This procedure is to be followed until end of Q2 2021 when we officially move the the new AWS account. Following the move, TSC will assist us in creating the user accounts, as we would not be allowed do it ourselves. CD will provide an efficient means to do it.

**Comment by mshadbolt** [comment]
wranglers don't have access to the AWS console (or if we do I don't know how to). I am still not clear on whether wranglers can run these commands or we will need to ask a dev to run the commands to generate the keys.

**Comment by lauraclarke** [comment]
I agree with @mshadbolt that these dont look like commands that the wranglers have the capacity to run. If they can't and should be able to we need to fix this, if they shouldn't be able to we need to understand who they need to request it from and by what means

I thought CD was the only person who can create the accounts, what notice do the cloud consultants need to be sent about this or are they no longer part of the process

**Comment by amnonkhen** [comment]
Indeed Wranglers cannot currently create aws users. One of the devs would take care of it until the end of Q2, when TSC would take the reigns.

**Comment by clairerye** [comment]
Can we create credentials in advance? If we need to wait for tsc to do them when I request comes in from a contributor I worry that we won't be able to give the contributor access credentials in the same working day as we would now.

**Comment by amnonkhen** [comment]
What is a reasonable response time for Contributors? Same working day?
I will make sure with CD that this can be achieved.

File: docs/SOPs/dataset_wrangling_SOP.md [hunk 1]
```diff
@@ -120,9 +120,21 @@ After the spreadsheet is generated some manual steps can help contributors under
...
-## Raw data upload (fastq)
+## Raw Data Upload (fastq)
 
-In order for a contributor to upload their data, you will need to provide them with a data upload area UUID as well as a set of contributor AWS access keys. 
+In order for a contributor to upload their data, you will need to:
+
+- provide them with a data upload area UUID
+- create an AWS user for them
+  
+```shell
```
Comment (ESapenaVentura) [inline docs/SOPs/dataset_wrangling_SOP.md:130]: It should probably state that a wrangler can't run this process on its own, so we need to come to a dev to do this

**Comment by mshadbolt** [comment]
If wranglers can't run the lines of code I don't think the wrangling SOP is the best place to put them.

I would prefer some text like:
Wranglers should request a new contributor account from the dev on operations via slack/github/email. The wrangler needs to provide the following information when requesting a new user account. <insert whatever information we need to provide>

**Comment by amnonkhen** [comment]
Sounds fair, I will add this instruction, but leave the commands here as well.

**Review by ami-day — APPROVED** [review]
This looks clear to me. So, as a wrangler, we will always contact a dev to ask them to do this? I don't need to be able to create an account, do I?

**Pull request merged**